app/views/static/_home_cover.html.erb

```diff
@@ -0,0 +1,68 @@
+<div class="cover">
+  <div class="cover-image" style="background-image: url(<%= asset_path 'static/home-cover-04.jpg' %>)"></div>
+  <div class="container">
+    <div class="row">
+      <div class="col-xs-12 col-sm-7">
+        <div class="cover-text">
+          <h1 class="cover-text__top">Discover the greatest works of art ever painted.</h1>
+          <h2 class="cover-text__bottom">Supercharge your study session with curated lesson plans designed to help you learn things faster and remember them even longer.</h2>
+        </div>
+      </div>
+      <div class="visible-xs col-xs-12 signup-form">
+        <div class="form-group">
+          <button type="submit" class="btn btn-default btn-block">Signup for Free</button>
+        </div>
+        <div calss="form-group">
+          <div class="hr-label">
+            <span class="hr-label__text">or</span>
+          </div>
+        </div>
+        <div class="form-group">
+          <button type="submit" class="btn btn-default btn-block">Signup for Free with Facebook</button>
+        </div>
+      </div>
+      <div class="hidden-xs col-xs-12 col-sm-5">
```

Comment:
Just noticed this hidden-xs class, thats cool, so you're actually swapping out HTML for mobile. In that case, we could easily hide the form on mobile and have two behaviors for the signup button depending on screen size. :+1: